packaging/centos/datadog-agent.init

```diff
@@ -150,6 +151,14 @@ info() {
     exit $(($FORWARDER_RETURN+$COLLECTOR_RETURN+DOGSTATSD_RETURN))
 }
 
+reload() {
+    kill -HUP `cat $COLLECTOR_PIDFILE` 2>/dev/null
+    if [ $? -ne 0 ]; then
+        kill -SIGHUP `cat $COLLECTOR_PIDFILE`
```

Comment:
Not the cleanest code we have, but that will work on platforms that only have one or the other. :+1: